### ISSUE TITLE
Rotation Augmentation

### ISSUE THREAD

**Issue by chandlerbing65nm** [google-research/vision_transformer#142]
Hello, I did not find in the paper and code anything related to rotation augmentation. Did you use rotation as one of the augmentations?

**Comment by andsteing** [comment]
No, we only used Inceptino-style random image crop (i.e. size + aspect ration are random; see [GoogLeNet paper](https://www.cs.unc.edu/~wliu/papers/GoogLeNet.pdf) for details) and random left/right flip.
